### PR TITLE
Auth category

### PR DESCRIPTION
**Pull request by TrekSoft**
Bringing the current auth work over to Master.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: aws-auth-cognito/README.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
+## User Acceptance Testing Guide
```
Comment (jpignata) [inline aws-auth-cognito/README.md:1]: Maybe yank this before merging? This was super helpful and thank you!

File: aws-auth-cognito/gradle.properties [hunk 2]
```diff
@@ -0,0 +1,4 @@
+POM_ARTIFACT_ID=aws-auth-cognito
+POM_NAME=Amplify Framework for Android - Auth
+POM_DESCRIPTION=Amplify Framework for Android - Auth Plugin for Amazon Cognito (Preview)
```
Comment (jpignata) [inline aws-auth-cognito/gradle.properties:3]: We should rip these Preview descriptions out now.

File: aws-auth-cognito/README.md (hunk 1 above)
Comment (jamesonwilliams) [inline aws-auth-cognito/README.md:1]: This is a weird way to start the plugin README.

File: aws-auth-cognito/proguard-rules.pro [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# Add project specific ProGuard rules here.
```
Comment (jamesonwilliams) [inline aws-auth-cognito/proguard-rules.pro:1]: The GitHub rendering of the `consumer-rules.pro` says "No changes." Is that file included, like the other plugins have?

Comment (TrekSoft) [reply aws-auth-cognito/proguard-rules.pro:1]: It was included but only had the original default comment, not the content like the others had so I went ahead and added the same content as was in the data store one.

File: aws-auth-cognito/README.md (hunk 1 above)
Comment (TrekSoft) [reply aws-auth-cognito/README.md:1]: Yeah this was just for John Pignata during development - removing

File: aws-auth-cognito/gradle.properties (hunk 2 above)
Comment (TrekSoft) [reply aws-auth-cognito/gradle.properties:3]: Done - took them out of all the categories

File: aws-auth-cognito/README.md (hunk 1 above)
Comment (TrekSoft) [reply aws-auth-cognito/README.md]: Done

**Pull request merged**